src/actions/actions.ts

```diff
@@ -5132,6 +5133,32 @@ class MoveFormat extends BaseMovement {
 }
 
 @RegisterAction
+class MoveUpercaseLine extends BaseMovement {
```

Comment:
Can you leave a comment here saying that these are for guu and gUU (for searchability mostly)